recipes/buffer-atob-btoa/src/workflow.ts

```diff
@@ -0,0 +1,57 @@
+import type { Edit, Range, SgRoot } from '@codemod.com/jssg-types/main';
+import type Js from '@codemod.com/jssg-types/langs/javascript';
+import { getNodeRequireCalls } from '@nodejs/codemod-utils/ast-grep/require-call';
+import { getNodeImportStatements } from '@nodejs/codemod-utils/ast-grep/import-statement';
+import { removeLines } from '@nodejs/codemod-utils/ast-grep/remove-lines';
+
+export default function transform(root: SgRoot<Js>): string | null {
+  const rootNode = root.root()
+  const edits: Edit[] = [];
+  const linesToRemove: Range[] = [];
+
+  const requireStatements = getNodeRequireCalls(root, 'buffer');
+  const importStatements = getNodeImportStatements(root, 'buffer');
+  const atobFunctionCalls = rootNode.findAll({
+    rule: {
+      pattern: 'buffer.atob($ARG)'
+    }
+  });
+
+  // Remove all buffer require statements
+  for (const statement of requireStatements) {
+    linesToRemove.push(statement.range());
+  }
+
+  // Remove all buffer import statements
+  for (const statement of importStatements) {
+    linesToRemove.push(statement.range());
+  }
```

Comment:
Before remove imports/requires to `node:buffer` is necessary, if this is not used in other cases.

So, just remove the entire line only if it is not used for other scenarios by example

```js
import Buffer from "node:buffer"

Buffer.isUtf8("test") // cannot be removed because Buffer is used for other usecase
```

When destructuring exists, just remove the `atob` function.

**Before**
```js
import { atob, isUtf8 } from "node:buffer"
```

**After**
```js
import { isUtf8 } from "node:buffer"
```

You can use `utils/ast-grep/remove-binding.ts` to help in this scenario, you just check first if it is not used in other places before try remove it.